src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const result of actions) {
+    switch (true) {
+      case (result.type === 'addProperties'):
+        const mas = Object.keys(result.extraData);
+
+        for (const key of mas) {
+          state[key] = result.extraData[key];
+        }
+        break;
+      case (result.type === 'removeProperties'):
+        for (const key of result.keysToRemove) {
+          delete state[key];
+        }
+        break;
+      case (result.type === 'clear'):
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+    }
+  }
```

Comment:
switch(true) is useless) Use switch(result.type) so you will be able to use cases in proper way `case 'clear': 
some code`